subprojects/core/src/integTest/groovy/org/gradle/api/tasks/IncrementalInputsIntegrationTest.groovy

```diff
@@ -229,10 +229,11 @@ class IncrementalInputsIntegrationTest extends AbstractIncrementalTasksIntegrati
         """
 
         file("input").createDir()
+        def inputPath = file('input').absolutePath
 
         expect:
         fails("myTask")
-        failureHasCause("Multiple entries with same key: ${file('input').absolutePath}=inputTwo and ${file('input').absolutePath}=inputOne")
+        failureHasCause("Multiple entries with same value: inputTwo=$inputPath and inputOne=$inputPath")
```

Comment:
Did the exception for duplicate values in the builder change?

Reply:
Yes. I think the new one is actually more correct. But I guess we shouldn’t be checking for the text of an exception that is not emitted by us in the first place. If this is an important use case, we should print a clear message of our own here.

Reply:
Ah! Since it's a compilation failure, this isn't going to sneak up and bite you at runtime.. Good to know!